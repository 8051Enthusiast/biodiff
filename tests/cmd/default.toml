bin.name = "biodiff"
args = ["--color", "--config", "tests/cmd/default.json", "--print", "-gglobal", "tests/text_x.txt", "tests/text_y.txt"]
status.code = 0