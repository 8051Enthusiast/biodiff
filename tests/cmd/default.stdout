[38;5;8m 0000 [39m54 68 69 73 20 69 73 20  73 6f 6d 65 20 63 6f 6d [38;5;8m | 0000 [39m54 68 69 73 20 69 73 20  73 6f 6d 65 20 63 6f 6d 
[38;5;8m 0010 [39m6d 6f 6e 20 74 65 78 74  20 62 65 74 77 65 65 6e [38;5;8m | 0010 [39m6d 6f 6e 20 74 65 78 74  20 62 65 74 77 65 65 6e 
[38;5;8m 0020 [39m20 62 6f 74 68 20 66 69  6c 65 73 2e [38;5;10m20 54 68 69 [38;5;8m | 0020 [39m20 62 6f 74 68 20 66 69  6c 65 73 2e [38;5;10m            [39m
[38;5;8m 0030 [38;5;10m73 20 74 65 78 74 20 69  73 20 6f 6e 6c 79 20 69 [38;5;8m | 002c [38;5;10m                                                 [39m
[38;5;8m 0040 [38;5;10m6e 20 74 68 69 73 20 66  69 72 73 74 20 66 69 6c [38;5;8m | 002c [38;5;10m                                                 [39m
[38;5;8m 0050 [38;5;10m65 2e [39m0a 54 68 65 20 63  68 61 72 61 63 74 65 72 [38;5;8m | 002c [38;5;10m      [39m0a 54 68 65 20 63  68 61 72 61 63 74 65 72 
[38;5;8m 0060 [39m20 69 73 20 [38;5;9m58 [39m20 69 6e  73 74 65 61 64 20 6f 66 [38;5;8m | 003a [39m20 69 73 20 [38;5;9m59 [39m20 69 6e  73 74 65 61 64 20 6f 66 
[38;5;8m 0070 [39m20 [38;5;9m59 [39m2e 0a 53 6f 6d 65  20 6d 6f 72 65 20 74 65 [38;5;8m | 004a [39m20 [38;5;9m58 [39m2e 0a 53 6f 6d 65  20 6d 6f 72 65 20 74 65 
[38;5;8m 0080 [39m78 74 20 74 6f 20 6d 61  6b 65 20 73 75 72 65 20 [38;5;8m | 005a [39m78 74 20 74 6f 20 6d 61  6b 65 20 73 75 72 65 20 
[38;5;8m 0090 [39m74 68 65 20 6e 75 6d 62  65 72 20 6f 66 20 73 61 [38;5;8m | 006a [39m74 68 65 20 6e 75 6d 62  65 72 20 6f 66 20 73 61 
[38;5;8m 00a0 [39m6d 65 20 63 68 61 72 61  63 74 65 72 73 20 62 65 [38;5;8m | 007a [39m6d 65 20 63 68 61 72 61  63 74 65 72 73 20 62 65 
[38;5;8m 00b0 [39m74 77 65 65 6e 20 74 68  65 20 74 65 78 74 73 20 [38;5;8m | 008a [39m74 77 65 65 6e 20 74 68  65 20 74 65 78 74 73 20 
[38;5;8m 00c0 [39m69 73 20 73 75 66 66 69  63 69 65 6e 74 6c 79 20 [38;5;8m | 009a [39m69 73 20 73 75 66 66 69  63 69 65 6e 74 6c 79 20 
[38;5;8m 00d0 [39m68 69 67 68 2e 0a [38;5;10m                               [38;5;8m | 00aa [39m68 69 67 68 2e 0a [38;5;10m54 68  69 73 20 74 65 78 74 20 [39m
[38;5;8m 00d6 [38;5;10m                                                 [38;5;8m | 00ba [38;5;10m69 73 20 6f 6e 6c 79 20  69 6e 20 74 68 69 73 20 [39m
[38;5;8m 00d6 [38;5;10m                                        [39m48 65 72 [38;5;8m | 00ca [38;5;10m73 65 63 6f 6e 64 20 66  69 6c 65 2e 20 [39m48 65 72 
[38;5;8m 00d9 [39m65 20 69 73 20 73 6f 6d  65 20 6d 6f 72 65 20 63 [38;5;8m | 00da [39m65 20 69 73 20 73 6f 6d  65 20 6d 6f 72 65 20 63 
[38;5;8m 00e9 [39m6f 6d 6d 6f 6e 20 74 65  78 74 20 74 6f 20 77 72 [38;5;8m | 00ea [39m6f 6d 6d 6f 6e 20 74 65  78 74 20 74 6f 20 77 72 
[38;5;8m 00f9 [39m61 70 20 69 74 20 75 70  2e [38;5;10m                     [38;5;8m | 00fa [39m61 70 20 69 74 20 75 70  2e [38;5;10m                     [39m
